docs/vectorizer-quick-start.md

```diff
@@ -90,8 +90,8 @@ Now we can create and run a vectorizer. A vectorizer is a pgai concept, it proce
     SELECT ai.create_vectorizer(
          'blog'::regclass,
          loading => ai.loading_column('contents'),
-         destination => 'blog_contents_embeddings',
-         embedding => ai.embedding_ollama('nomic-embed-text', 768),
+         destination => ai.destination('blog_contents_embeddings'),
```

Comment:
Isn't `destination_column` the name of the function 🤔 ?
```suggestion
         destination => ai.destination_column('blog_contents_embeddings'),
```

Reply:
that's a good catch I did a faulty find and replace 😓 thank you!

Reply:
`ai.destination_table` is the function name that's equivalent to what we had before :D

Reply:
> `ai.destination_table` is the function name that's equivalent to what we had before :D

that's a good catch I did a faulty find ~and replace~ 😓 thank you!

😆